### DISCUSSION TITLE
Monika bday kiss

### DISCUSSION THREAD

**Discussion by AkiStar2** [Monika-After-Story/MonikaModDev#7872] [QA - tech support + general] [answered]
i heard monika kisses you first time at special ocasion if you have affection 400+ i have 439 affection i threw birthday party at her she just puted the cake away and started acting normal what should i do i even said her happy birthday in the section ''iwant to tell you something''

**Comment by lunulae** [comment, marked as answer]
I believe the kiss should happen closer to the end of the day, when she thanks you for spending it with her.
